### PR TITLE
update references to policy commands

### PR DESCRIPTION
**Pull request by deads2k**
Minor updates for consistency of cluster policy in comments, help, and readmes.

@liggitt ptal

### CODE REVIEW

File: images/ipfailover/keepalived/README.md [hunk 1]
```diff
@@ -67,12 +67,12 @@ Pre-requisites/Prep Time
 
         $ vagrant ssh minion-1
         #  Add user and project.
-        openshift ex policy add-user view anypassword:test-admin
-        openshift ex new-project test --display-name="Failover Sample" \
-           --description="Router Failover" --admin=anypassword:test-admin
+        osadm policy add-role-to-user view test-admin
```
Comment (liggitt) [inline images/ipfailover/keepalived/README.md]: what namespace is this supposed to be adding view rights for? seems like it's not needed?

**Comment by liggitt** [comment]
can we replace ": empty means 'ClusterRole'" with "; empty means a role defined in cluster policy" in modify_roles.go?

File: images/ipfailover/keepalived/README.md (hunk 1 above)
Comment (deads2k) [inline images/ipfailover/keepalived/README.md]: it's not required.  I have no idea what the intent of this doc was.  It had rotted.

Comment (deads2k) [inline images/ipfailover/keepalived/README.md]: Looks like default namespace

**Comment by deads2k** [comment]
text updated.

**Comment by liggitt** [comment]
[merge]

**Pull request merged**